Rendera/src/main.cpp

```diff
@@ -1,6 +1,96 @@
-#include <iostream>
-#include <xsimd/xsimd.hpp>
+#include "../include/vec3.hpp"
```

Comment:
The include paths need not have `../include` in them, because that is taken care of by the `target_include_directories` command in the `CMakeLists.txt`, which tells CMake where to look for the header files (which it then passes on to the compiler).

This should say `#include "vec3.hpp"`, and the other includes should be changed accordingly.